src/wymeditor/editor/undo-redo.js

```diff
@@ -86,6 +86,12 @@ WYMeditor.UndoRedo.prototype._add = function () {
         return;
     }
 
+    if (wym.ih._isResizeHandleAttached()) {
+        // so that the resize handle
+        // does not end up as content in the history
+        wym.ih._detachResizeHandle();
```

Comment:
Does this mean that if someone clicks to resize the handler, then clicks somewhere else, then resize handler will poof? Is that OK?